pyaedt/modules/CableModeling.py

```diff
@@ -699,16 +699,20 @@ def _init_from_json(self, json_file_name):
             json_dict = json_to_dict(json_file_name)
 
         # Cable implementation
-        if json_dict["Add_Cable"].lower() == "true":
+        if json_dict["Add_Cable"].lower() == "true" or json_dict["Update_Cable"].lower() == "true":
             try:
                 if json_dict["Cable_prop"]["CableType"].lower() in ["bundle", "straight wire", "twisted pair"]:
                     self.cable_type = json_dict["Cable_prop"]["CableType"]
                 else:
-                    msg = "Cable type is not valid. Available values are: bundle, straight wire, twisted pair"
+                    msg = "Cable type is not valid. Available values are: bundle, straight wire, twisted pair."
                     raise ValueError(msg)
 
-                if json_dict["Update_Cable"]:
-                    self.updated_name = json_dict["Cable_prop"]["UpdatedName"]
+                if json_dict["Update_Cable"].lower() == "true":
+                    if json_dict["Cable_prop"]["UpdatedName"]:
+                        self.updated_name = json_dict["Cable_prop"]["UpdatedName"]
+                    else:
+                        msg = "Insert a valid updated name for cable."
+                        raise ValueError(msg)
```

Comment:
If the variable is used only once, it is not necessary to create one. 
```suggestion
                        raise ValueError("Insert a valid updated name for cable.")
```